e2e/cypress/integration/multi-day-export.js

```diff
@@ -26,6 +25,7 @@ describe('Multi day export', () => {
   it('Allows you to download API splits using the API splits dialog', () => {
     cy
```

Comment:
should this need the `asATestSetupUser` role to be able to add a flight?

Reply:
adding flights actually doesn't need any roles because it's only on the Test Controller which has no auth protection